src/components/ListingDetails/index.tsx

```diff
@@ -0,0 +1,320 @@
+/* eslint-disable react/jsx-no-comment-textnodes */
+import React, { useContext, useMemo } from 'react';
+import {
+  // timestampStringToDate,
+  // parseDate,
+  formatTimestamp,
+  parseAgency,
+  parseTimeCommitment,
+} from '@/utils/helpers';
+import { H2, H3, H4, P, StrongP } from '@/styles/text';
+import {
+  CaseListing,
+  DocumentTranslation,
+  Interpretation,
+  LimitedCaseAssignment,
+  Listing,
+} from '@/types/schema';
+import { ProfileContext } from '@/utils/ProfileProvider';
+import COLORS from '@/styles/colors';
+// import InterestForm from '../InterestForm';
+import { Flex } from '@/styles/containers';
+import logo from '@/lib/dohs_logo.webp';
+import { LinkButton } from '../Button';
+import Icon from '../Icon';
+import {
+  CaseDisplay,
+  InfoContainer,
+  FieldContainer,
+  InnerFieldContainer,
+  AuthButtons,
+  IconTextGroup,
+  BorderedSection,
+  // InlineP
+  DateText,
+} from './styles';
+
+const renderField = (label: string, value: string | boolean | number) => (
+  <FieldContainer>
+    <StrongP>{label}</StrongP>
+    <P>{value}</P>
+  </FieldContainer>
+);
+
+const caseFields = [
+  // Relief sought
+  {
+    label: 'Relief Sought',
+    getValue: (data: Listing) =>
+      (data as CaseListing).relief_codes.join(', ') || 'N/A',
+  },
+  // Time Commitment
+  {
+    label: 'Time Commitment',
+    getValue: (data: Listing) =>
+      parseTimeCommitment(
+        (data as CaseListing).hours_per_week,
+        (data as CaseListing).num_weeks,
+      ),
+  },
+  // Remote/In Person
+  {
+    label: 'Remote/In Person',
+    getValue: (data: Listing) =>
+      (data as CaseListing).is_remote ? 'Remote' : 'In Person',
+  },
+  // Adjudicating Agency
+  {
+    label: 'Adjudicating Agency',
+    getValue: (data: Listing) => {
+      const d = data as CaseListing;
+      return d.adjudicating_agency ? parseAgency(d.adjudicating_agency) : 'N/A';
+      // (data as CaseListing).adjudicating_agency ? parseAgency(data.adjudicating_agency) : 'N/A',
+    },
+  },
+  // Client Languages
+  {
+    label: 'Client Language(s)',
+    getValue: (data: Listing) => (data as CaseListing).languages.join(', '),
+  },
+  // Client Country of Origin
+  {
+    label: 'Client Country of Origin',
+    getValue: (data: Listing) => (data as CaseListing).country || 'N/A',
+  },
+  // Client Location
+  {
+    label: 'Client Location',
+    getValue: (data: Listing) => (data as CaseListing).client_location || 'N/A',
+  },
+  // Custody Location
+  {
+    // Temporary
+    label: 'Custody Location',
```

Comment:
could we go ahead and remove this field? I think we should either be showing a "Client Location" or "Custody Location" label but not both, depending on whether the case's `is_detained` field is false/true.